test/functional/test_helper.py

```diff
@@ -0,0 +1,15 @@
+def get_available_port_range(from_range, to_range):
+    """Returns available local port number.
+    """
+    if not isinstance(from_range, int) or not isinstance(to_range, int):
```

Comment:
explicit types verifications is not very pythonic

simply cast values to in in `range` call and let it throw if it is not possible